src/main/java/cz/muni/fi/pv286/mmt/model/ResultTree.java

```diff
@@ -0,0 +1,58 @@
+package cz.muni.fi.pv286.mmt.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * ResultTree structure is used for array parser.
+ */
+public class ResultTree {
+    private byte[] value;
+    String intermediate;
```

Comment:
**issue:** Private visibility modifier missing

Insert visibility modifier for this attribute